test/event_analytics_constants.ts

```diff
@@ -571,3 +571,6 @@ export const HORIZONTAL_BAR_TEST_VISUALIZATIONS_DATA = {
 export const EXPLORER_START_TIME = 'Aug 28, 2023 @ 20:00:00.406';
 
 export const EXPLORER_END_TIME = 'Aug 28, 2023 @ 20:00:00.408';
+
+export const EXPLORER_DATA_GRID_QUERY =
+  "source = opensearch_dashboards_sample_data_logs | where match(request,'filebeat')";
```

Comment:
unused?

Reply:
This is used in `public/components/event_analytics/explorer/__tests__/data_grid.test.tsx` as a specific raw query being passed into the data grid for testing.

Reply:
move to test files if it's only used by tests?

Reply:
Isn't this the test constant file? If not, where would it be?

Reply:
my bad, i misread the filename, you can ignore this comment..